seminar_registration_system/member_info_capture/views.py

```diff
@@ -1,25 +1,47 @@
 from django.shortcuts import render,redirect
-from .models import Member
+from member_info_capture.models import Member
+from django.urls import reverse
+from django.contrib import messages
+from django.http import HttpResponse
 
 def member_info_capture(request):
     if request.method == 'POST':
         student_type = request.POST.get('student_type')
         last_name = request.POST.get('last_name')
-        first_names = request.POST.get('first_name')
+        first_names = request.POST.get('first_names')
         personal_email = request.POST.get('personal_email')
         student_id = request.POST.get('student_id')
         residential_address = request.POST.get('residential_address')
         member = Member(
             student_type=student_type,
             last_name=last_name,
-            first_name=first_names,
+            first_names=first_names,
             personal_email=personal_email,
             student_id=student_id,
             residential_address=residential_address,
         )
         member.save()
-        return redirect('member_info_capture_success')
-    return render(request, 'member_info_capture/templates/member_info_capture.html')
+        messages.success(request, 'Member added successfully')
+        #Redirect to the admin page after saving member's information
+        return redirect(reverse('admin: member_info_capture_member_change', args=[member.id]))
+    #Handle GET requests
+    return redirect('member_info_capture')
+
+def find_user(request):
+    if request.method == 'POST':
+        user_number = request.POST.get('user_number')
+        try:
+            member = Member.objects.get(user_number=user_number)
+            return redirect(reverse('admin: member_info_capture_member_change', args=[member.id]))
+        except Member.DoesNotExist:
+            messages.error(request, 'Member with user number {} does not exist'.format(user_number))
+            return redirect('member_info_capture')
+        except ValueError:
```

Comment:
this is for section 2.1, so we want to implement a functionaly where users/students are checked to see if they've already captured their information in Section 1 of the system.If they have, you want to display a confirmation message along with their student number. If not, you want to direct them to section 1 for information capture